docs/src/pages/docs/en/audio-player.md

```diff
@@ -24,7 +24,7 @@ Since audio players don’t use the same visual layout as video players do, we c
 </media-controller>
 
 ```html
-<script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome@4/+esm"></script>
+<script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome@4.8.0/+esm"></script>
```

Comment:
these can src url changes can be reverted because https://cdn.jsdelivr.net/npm/media-chrome@4/+esm will return the latest v4. sometimes it takes a while before the jsdelivr cache is purged